src/gasman.c

```diff
@@ -2302,8 +2320,7 @@ UInt CollectBags (
 
     /* * * * * * * * * * * * * * * check phase * * * * * * * * * * * * * * */
     done = CollectBags_Check(size, nrBags);
-    if (done == 2)
-        return 0;
+    // done: 0 - finished, 1 - not finished, 2 - out of memory
```

Comment:
That's wrong, correct:
```suggestion
    // done: 0 - not finished, 1 - finished, 2 - out of memory
```
or even better:
```suggestion
    // done: 0 - not done, 1 - done with allocation, 2 - done because out of memory
```

But really, isn't the name `done` self-explanatory? In PR #3803 I refactored this loop as well, replacing the `goto` construct by a loop: `do { ... } while (!done)` which says it quite well, IMHO: "repeat while not done"